#35603 Handle output from source generator as input to analyzer

svick opened the pull request.

svick committed:
#35603 Handle output from source generator as input to analyzer

svick committed:
#35603 Fixed log messages

gfraiteur reviewed (changes requested):
The problem is already handled by `SourceGeneratorHelper` and the code that calls it. There is probably a bug there, but I suspect the fix should be close to these files rather than upstream.

svick committed:
Revert "#35603 Handle output from source generator as input to analyzer"

This reverts commit ec233c5b50e95cdb190c19489b84a7ae773d46f2.

# Conflicts:
#	Metalama.Framework.DesignTime/TheDiagnosticAnalyzer.cs

svick committed:
#35603 Fix detecting Metalama source generated code

svick commented:
@gfraiteur You're right. `SourceGeneratorHelper` expects a relative path, but in VS Code, the path is now absolute. I have reverted my earlier code and fixed `SourceGeneratorHelper`.

svick committed:
#35603 Improved performance
